### DISCUSSION TITLE
Unable to Compile the MEGA ELS code

### DISCUSSION THREAD

**Discussion by Lester444** [kachurovskiy/megaels#20] [Q&A]
So I downloaded all the files and saved them into a single folder. Indeed all the other __.ino files are now detected when I open Digital_Feed_7e2.ino.

Upon compiling I get this error message and I can not proceed further.
![Error Msg](https://user-images.githubusercontent.com/23709087/195446019-3f76cf58-fc42-453a-8528-a9e69720bfda.png)

**Comment by kachurovskiy** [comment]
Looks like you saved the entire webpage. Download just the project files - https://github.com/kachurovskiy/megaels/archive/refs/heads/main.zip

**Reply by Lester444** [reply]
That link helped me a lot, it compiles now. Thank you. :)
